src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -2,7 +2,7 @@ class Crystal::Doc::Generator
   getter program : Program
 
   @base_dir : String
-  @is_crystal_repo : Bool
+  property is_crystal_repo : Bool
```

Comment:
```suggestion
  getter is_crystal_repo : Bool
```
Should be enough?

Reply:
No, the setter is used in `generator_spec`